docs/tutorial.md

```diff
@@ -0,0 +1,557 @@
+
+!!! note
+    This is a simple example that uses the `addPresenter` and `addUi` functions. In a real app, you'd likely use a `Presenter.Factory` and `Ui.Factory` to create your presenters and UIs dynamically.
+
+Once you have this instance, you can plug it into `CircuitCompositionLocals` ([docs](https://slackhq.github.io/circuit/api/0.x/circuit-foundation/com.slack.circuit.foundation/-circuit-composition-locals.html)) and be on your way. This is usually a one-time setup in your application at its primary entry point.
+
+=== "Android"
+    ```kotlin title="MainActivity.kt"
+    class MainActivity {
+      override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val circuit = Circuit.Builder()
+          // ...
+          .build()
+    
+        setContent {
+          CircuitCompositionLocals(circuit) {
+            // ...
+          }
+        }
+      }
+    }
+    ```
+
+=== "Desktop"
+    ```kotlin title="main.kt"
+    fun main() {
+      val circuit = Circuit.Builder()
+        // ...
+        .build()
+
+      application {
+        Window(title = "Inbox", onCloseRequest = ::exitApplication) {
+          CircuitCompositionLocals(circuit) {
+            // ...
+          }
+        }
+      }
+    }
+    ```
+
+=== "JS"
+    ```kotlin title="main.kt"
+    fun main() {
+      val circuit = Circuit.Builder()
+        // ...
+        .build()
+
+      onWasmReady {
+        Window("Inbox") {
+          CircuitCompositionLocals(circuit) {
+            // ...
+          }
+        }
+      }
+    }
+    ```
+
+## `CircuitContent`
+
+`CircuitContent` ([docs](https://slackhq.github.io/circuit/api/0.x/circuit-foundation/com.slack.circuit.foundation/-circuit-content.html)) is a simple composable that takes a `Screen` and renders it.
+
+```kotlin title="CircuitContent"
+CircuitCompositionLocals(circuit) {
+  CircuitContent(InboxScreen)
+}
+```
+
+Under the hood, this instantiates the corresponding `Presenter` and `Ui` from the local `Circuit` instance and connects them together. All you need to do is pass in the `Screen` you want to render!
+
+This is the most basic way to render a `Screen`. These can be top-level UIs or nested within other UIs. You can even have multiple `CircuitContent` instances in the same composition.
+
+## Adding navigation to our app
+
+An architecture isn't complete without navigation. Circuit provides a simple navigation API that's focused around a simple `BackStack` ([docs](https://slackhq.github.io/circuit/api/0.x/backstack/com.slack.circuit.backstack/-back-stack/index.html)) that is navigated via a `Navigator` interface ([docs]()). In most cases, you can use the built-in `SaveableBackStack` implementation ([docs](https://slackhq.github.io/circuit/api/0.x/backstack/com.slack.circuit.backstack/-saveable-back-stack/index.html)), which is saved and restored in accordance with whatever the platform's `rememberSaveable` implementation is.
```

Comment:
nit: Think this should read "An app isn't complete without navigation". 
```suggestion
An app isn't complete without navigation. Circuit provides a simple navigation API that's focused around a simple `BackStack` ([docs](https://slackhq.github.io/circuit/api/0.x/backstack/com.slack.circuit.backstack/-back-stack/index.html)) that is navigated via a `Navigator` interface ([docs]()). In most cases, you can use the built-in `SaveableBackStack` implementation ([docs](https://slackhq.github.io/circuit/api/0.x/backstack/com.slack.circuit.backstack/-saveable-back-stack/index.html)), which is saved and restored in accordance with whatever the platform's `rememberSaveable` implementation is.
```

Reply:
camera app may disagree ;P. Maybe "app architecture" for both?